### PR TITLE
Added eksctl and aws cli, enabled install wizard

### LINKED ISSUE #109: eksctl and aws cli
As a user, I want to be able to create a cluster for my project automatically.
As a developer, I need to tools to automate the AWS cluster creation process.

### PR DESCRIPTION
**Pull request by veikkahonkanen**
Resolves #109

### CODE REVIEW

**Review by eetupur — APPROVED** [review]

File: src/commands/install.ts [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+  Software.JDK8,
+  Software.JDK11,
+  Software.Homebrew,
+  Software.Docker,
+  Software.Minikube,
+  Software.KubernetesCLI,
+  Software.Kustomize,
+  Software.EKSctl,
+  Software.AWSCLI
+];
```
Comment (jarinykanen) [inline src/commands/install.ts]: These can be removed (check next comment)

File: src/commands/install.ts [hunk 2]
```diff
@@ -15,7 +35,7 @@ async function action(args) {
       const softwareResult = await this.prompt({
         type: 'list',
         name: 'software',
-        choices: [ 'NodeJs', 'GitHub', 'Git CLI', 'Maven', 'Java Development Kit 8', 'Java Development Kit 11', 'Homebrew', 'Docker', 'Minikube', 'Kubernetes CLI', 'Kustomize' ],
+        choices: choices,
```
Comment (jarinykanen) [inline src/commands/install.ts]:
```suggestion
       choices: Object.keys(Software),
```

File: src/commands/install.ts [hunk 3]
```diff
@@ -15,7 +35,7 @@ async function action(args) {
       const softwareResult = await this.prompt({
         type: 'list',
         name: 'software',
```
Comment (jarinykanen) [inline src/commands/install.ts]:
```suggestion
        type: "list",
        name: "software",
```

File: src/commands/install.ts [hunk 4]
```diff
@@ -28,14 +48,36 @@ async function action(args) {
...
+
+    } else if (software == Software.KubernetesCLI) {
+      installCommands = await InstallUtils.installKubeCtl();
+
+    } else if (software == Software.EKSctl) {
+      installCommands = await InstallUtils.installEksctl();
+
+    } else {
+      installCommands = await InstallUtils.installSW(software);
+    }
```
Comment (jarinykanen) [inline src/commands/install.ts]: Replace with switch case

**Review by jarinykanen — CHANGES_REQUESTED** [review]

File: src/commands/install.ts (hunk 4 above)
Comment (veikkahonkanen) [reply src/commands/install.ts]: Replaced with switch case

**Review by eetupur — APPROVED** [review]

**Review by jarinykanen — APPROVED** [review]

**Pull request merged**